.github/workflows/audit.yml

```
@@ -21,9 +24,36 @@ jobs:
     permissions:
       contents: read
       issues: write
+      security-events: write
     steps:
       - uses: actions/checkout@v4
-      - uses: actions-rust-lang/audit@v1
-        name: Audit Rust dependencies
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)